benefits/pension_scheme.md

```diff
@@ -2,20 +2,30 @@
 
 We provide a pension scheme inline with the UK Government's rollout of automatic enrolment. We automatically enrol everyone in our pension scheme after their probation period (usually 3 months).
 
-- Our current pension plan is provided by Royal London
+- Our current pension plan is provided by [Royal London](https://www.royallondon.com/)
 - If you want to opt out there are now [Government guidelines](https://www.gov.uk/workplace-pensions/if-you-want-to-leave-your-workplace-pension-scheme) on doing so
 - You can opt to contribute more to your employee contribution, feel free to speak to finance@madetech.com about this
 - You can read more about Workplace pensions on [gov.uk](https://www.gov.uk/workplace-pensions/about-workplace-pensions)
 - You can read more about Pensions and some third party advice on [MoneySavingExpert](http://www.moneysavingexpert.com/savings/discount-pensions)
 
 ## Pension contributions
 
-As your employer we will contribute towards your pension alongside your own contribution. Below are the guidelines to the contributions we pay, and the minimum contribution you can pay. We will scale up contributions over the next 3 years, subject to parliamentary approval and guidelines.
+As your employer we will contribute towards your pension alongside your own contribution. 
 
-When | We pay | You pay | Total minimum contribution
------|--------|---------|----------------------------
-Now | 1% | 1% | 2%
-From April 2018 | 2% | 3% | 5%
-From April 2019 | 3% | 5% | 8%
+From April 2019 the minimum amount that you pay into your pension is **5%** and we pay **3%**—a total minimum contribution equivilent to **8%** of your salary. 
 
 You can choose to increase your pension contribution sooner. You can do this by speaking with our finance department on finance@madetech.com.
+
+## How contributions are calculated
+
+The workplace pension plan we use is a **relief at source** scheme. This means that pension contributions are taken _after_ tax has been deducted from your salary. 
+
+As private pension contributions are tax-exempt, the income tax that has been paid on your contributions is claimed back when the money reaches your pension plan. When you choose to pay a certain amount of your monthly salary into the pension plan, 80% of that amount will be deducted from you net pay and the remaining 20% (representing the [basic rate of income tax](https://www.gov.uk/income-tax-rates)) will be claimed back by Royal London as tax relief.
+
+### Higher rate tax payer
+
+If you are a higher rate tax payer, then you pay 40% income tax on any earnings above £50,000, leaving you entitled to an additional tax relief beyond the 20% recovered automatically. You must claim this additional tax relief to ensure that your pension contributions are entirely tax-free.
```

Comment:
I tried to ignore this but I can't :-( I think the 'then' is superfluous on the first line:

Consider: If you are a higher rate tax payer you pay 40% income tax on any
Over: If you are a higher rate tax payer, then you pay 40% income tax on any